core/src/main/java/inetsoft/web/portal/controller/RepositoryTreeController.java

```diff
@@ -201,7 +201,7 @@ public TreeNodeModel[] getChildrenNodes(String path, String permission, int sele
             continue;
          }
 
-         entry.setFavoritesUser(repositoryTreeService.hasFavoritesUser(entry, principal));
+         entry.setFavoritesUser(!isDefaultOrgAsset && repositoryTreeService.hasFavoritesUser(entry, principal));
```

Comment:
Does this also affect users of the host org? If that's the case, then it's incorrect.